sphere_grid_reader.py

```diff
@@ -0,0 +1,32 @@
+# Libraries and Core Files
+import memory.main
+from memory.sphere_grid import SphereNodeType, sphere_grid
+
+# Wait for memory to finish loading
+while not memory.main.start():
+    pass
+
+
+# Example code to read a particular node
+node_123 = sphere_grid.get_node_at(123)
+
+print(f"Node at idx=123 is {node_123}")
+
+last_node = 0
+while True:
```

Comment:
Do we want this to actually run forever? Because this doesn't have a break or a return.

Reply:
See comment below.